BUAN and RB Documentation update (#2278)
dipy/dipy: doc/interfaces/buan_flow.rst

folder you downloaded from streamline atlas link and ``subjects_small`` folder
downloaded from test data link.

> Where, ``bundles`` folder had following nodel bundles::

Reviewer: Why past tense?

> Where, ``bundles`` folder has following nodel bundles::

Reviewer 2: ```suggestion
Where, ``bundles`` folder has following model bundles::
```

    subjects_small
    ├── control
    │   ├── 3805
    │   │   ├── anatomical_measures
    │   │   ├── org_bundles
    │   │   └── rec_bundles
    │   ├── 3806
    │   │   ├── anatomical_measures
    │   │   ├── org_bundles
    │   │   └── rec_bundles
    │   ├── 3809
    │   │   ├── anatomical_measures
    │   │   ├── org_bundles
    │   │   └── rec_bundles
    │   ├── 3850
    │   │   ├── anatomical_measures
    │   │   ├── org_bundles
    │   │   └── rec_bundles
    │   └── 3851
    │       ├── anatomical_measures
    │       ├── org_bundles
    │       └── rec_bundles
    └── patient
        ├── 3383
        │   ├── anatomical_measures
        │   ├── org_bundles
        │   └── rec_bundles
        ├── 3385
        │   ├── anatomical_measures
        │   ├── org_bundles
        │   └── rec_bundles
        ├── 3387
        │   ├── anatomical_measures
        │   ├── org_bundles
        │   └── rec_bundles
        ├── 3392
        │   ├── anatomical_measures
        │   ├── org_bundles
        │   └── rec_bundles
        └── 3552
            ├── anatomical_measures
            ├── org_bundles
            └── rec_bundles

> And each subject has following directory structure::

Reviewer: English

Run the following commandline for visualizing group differences on the model bundle::

    dipy_horizon bundles/AF_L.trk lmm_plots/AF_L_fa_pvalues.npy --buan --buan_thr 0.05

Where, ``AF_L.trk `` is located in your model bundle folder ``bundles`` and
``AF_L_fa_pvalues.npy`` is saved in output folder ``lmm_plots`` of dipy_buan_lmm workflow

Output of this commandline is interactive visualization window. Example snapshot:

.. figure:: https://github.com/dipy/dipy_data/blob/master/AF_L_highlighted.png?raw=true
    :width: 70 %
    :alt: alternate text
    :align: center

    Result plot for left arcuate fasciculus (AF_L) with highlighted group differences
    area in red color on the bundle where pvalues < 0.05.

Let's use a different highlight color this time on ``CST_L`` bundle::

     dipy_horizon bundles/CST_L.trk lmm_plots/CST_L_fa_pvalues.npy --buan --buan_thr 0.05 --buan_highlight 1 1 0

.. figure:: https://github.com/dipy/dipy_data/blob/master/CST_L_highlighted.png?raw=true
    :width: 50 %
    :alt: alternate text
    :align: center

    Result plot for left corticospinal tract left (CST_L) with highlighted group differences

>     area in yellow color on the bundle where pvalues < 0.05.

Reviewer: Improve flow. Possibly break in two sentences.

    Result plot for left arcuate fasciculus (AF_L) on FA measure

We can also visualize and highlight the specific location of group differences on the bundle by providing

> output pvalues file from dipy_buan_lmm workflow. User can specify at what level of

Reviewer 2: ```suggestion
output p-values file from dipy_buan_lmm workflow. The user can specify at what level of
```

> significance they want to see group differences by providing threshold value of pvalue to ``buan_thr`` (default 0.05).

Reviewer 2: ```suggestion
significance they want to see group differences by providing threshold value of p-value to ``buan_thr`` (default 0.05).
```

The color of the highlighted area can be specified by providing RGB color values to ``buan_highlight`` (Default Red)

Run the following commandline for visualizing group differences on the model bundle::

    dipy_horizon bundles/AF_L.trk lmm_plots/AF_L_fa_pvalues.npy --buan --buan_thr 0.05

Where, ``AF_L.trk `` is located in your model bundle folder ``bundles`` and
``AF_L_fa_pvalues.npy`` is saved in output folder ``lmm_plots`` of dipy_buan_lmm workflow

> Output of this commandline is interactive visualization window. Example snapshot:

Reviewer 2: ```suggestion
The output of this commandline is an interactive visualization window. Example snapshot:
```
